config/application.rb

```diff
@@ -45,33 +45,23 @@ class DavidRunger::Application < Rails::Application
   # ActiveJob/Sidekiq
   config.active_job.queue_adapter = :sidekiq
 
-  # ActionMailer
-  config.action_mailer.default_url_options =
-    case Rails.env
-    when 'production'
-      # :nocov:
-      { host: DavidRunger::CANONICAL_DOMAIN, protocol: 'https' }
-      # :nocov:
-    else
-      { host: 'localhost:3000', protocol: 'http' }
-    end
```

Comment:
This has been moved to the individual environment configs. In test, I think we already had one, which I guess must have been overriding this.